src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -84,6 +84,28 @@ public sealed class CompletionItem : IComparable<CompletionItem>
         /// </summary>
         internal Document Document { get; set; }
 
+        /// <summary>
+        /// The name of the <see cref="CompletionProvider"/> that created this 
+        /// <see cref="CompletionItem"/>. Not available to clients. Only used by 
+        /// the Completion subsystem itself for things like getting description text
+        /// and making additional change during commit.
+        /// </summary>
+        internal string ProviderName { get; set; }
+
+        /// <summary>
+        /// Whether to cache converted completion item for target editor. If returns <see langword="true"/>,
+        /// <see cref="CachedEditorCompletionItem"/> is used for storing the item.
+        /// Not available to clients. Only used to improve perf by Completion subsystem on a case-by-case basis.
+        /// </summary>
+        internal bool UseEditorCompletionItemCache { get; }
```

Comment:
as per above, i think we can remove this.

Reply:
fixed